README.md

```diff
@@ -1,2 +1,52 @@
 # wcd2reshare
 redirect from Worldcat Discovery to BorrowDirect / VuFind
+
+## Required ENV
+`SENTRY_DSN` = If set to a valid Sentry DSN, enables Sentry exception monitoring. This is not needed for local development.
+
+`WORKSPACE` = Set to `dev` for local development, this will be set to `stage` and `prod` in those environments by Terraform.
+
+## Developing locally
+
+<https://docs.aws.amazon.com/lambda/latest/dg/images-test.html>
+
+### Makefile commands for installation and dependency management
+
+```bash
+make install # installs with dev dependencies
+make test # runs tests and outputs coverage report
+make lint # runs code linting, quality, and safety checks
+make update # updates dependencies
+```
+
+### Build the container
+
+```bash
+make dist-dev
+```
+
+### Run the default handler for the container 
+
+```bash
+docker run -p 9000:8080 wcd2reshare-dev:latest
```

Comment:
One more teensy change: this won't work without passing the required env `-e WORKSPACE=dev` to the docker run command so I would add that here.